bcftbx/cmdparse.py

```diff
@@ -140,14 +145,14 @@ def parse_args(self,argv=None):
         Returns:
           A tuple of (cmd,options,args) where 'cmd' is the
           command, and 'options' and 'args' are the options and
-          arguments as returned by OptionParser.parse_args.
+          arguments as returned by ArgumentParser.parse_known_args.
```

Comment:
Update to indicate that the return value is dependent on the subparser being used.